server/routerlicious/packages/routerlicious-base/src/utils/sessionHelper.ts

```diff
@@ -284,10 +285,38 @@ export async function getSession(
 	clusterDrainingChecker?: IClusterDrainingChecker,
 	ephemeralDocumentTTLSec?: number,
 	connectionTrace?: StageTrace<string>,
+	readDocumentRetryTimeout: number = 150,
```

Comment:
`readDocumentRetryDelay` would be more accurate (in all places that timeout is used for this variable and config)